lib/generators/decidim/install_generator.rb

```diff
@@ -116,10 +118,20 @@ def letter_opener_web
       private
 
       def recreate_db
-        rails_command "db:environment:set db:drop" unless ENV["CI"]
-        rails_command "db:create"
-        rails_command "db:migrate"
-        rails_command "db:test:prepare"
+        rails "db:environment:set", "db:drop" unless ENV["CI"]
+        rails "db:create"
+
+        if options[:seed_db]
+          rails "db:migrate", "db:seed", env: "development"
+        else
+          rails "db:migrate", env: "development"
+        end
+
+        rails "db:migrate", env: "test"
```

Comment:
`rake db:test:prepare` loads the DB schema to the test DB, migrating will take longer :/ Is it the only way we have to test the migrations?

Reply:
Yup, this is a bit slower, but it's not the bottleneck. Our migrations are pretty fast at the moment. The reeeeeeally slow part is seeding...

However, this change is not really necessary. I'll remove it!